layers/raw_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Holger Hans Peter Freyther, Google, Inc. All rights reserved.
```

Comment:
Looking through the CLA, you're absolutely right.  The CLA allows us as Google to use the copyright code, but doesn't transfer the copyright to us as part of the contribution.

Would you mind, for this file, using the following language instead, and adding your name to AUTHORS?:

```
// Copyright 2014 The GoPacket Authors
```

I should probably standardize that on all the other files as well.